sdk/python/feast/expediagroup/vectordb/milvus_online_store.py

```diff
@@ -157,63 +157,95 @@ def update(
     def teardown(
         self,
         config: RepoConfig,
-        tables: Sequence[VectorFeatureView],
+        tables: Sequence[FeatureView],
         entities: Sequence[Entity],
     ):
         pass
 
     def _convert_featureview_schema_to_milvus_readable(
-        self, feast_schema: List[Field], vector_field, vector_field_dimensions
+        self, feast_schema: List[Field]
     ) -> CollectionSchema:
         """
         Converting a schema understood by Feast to a schema that is readable by Milvus so that it
         can be used when a collection is created in Milvus.
 
         Parameters:
-        feast_schema (List[Field]): Schema stored in VectorFeatureView.
+        feast_schema (List[Field]): Schema stored in FeatureView.
 
         Returns:
         (CollectionSchema): Schema readable by Milvus.
 
         """
         boolean_mapping_from_string = {"True": True, "False": False}
         field_list = []
-        dimension = None
 
         for field in feast_schema:
-            if field.name == vector_field:
-                field_name = vector_field
-                dimension = vector_field_dimensions
-            else:
-                field_name = field.name
 
+            field_name = field.name
             data_type = self._feast_to_milvus_data_type(field.dtype)
+            is_vector = False
+            dimensions = 0
+
+            if self._data_type_is_supported_vector(data_type):
+                is_vector = True
 
             if field.tags:
                 description = field.tags.get("description", " ")
                 is_primary = boolean_mapping_from_string.get(
                     field.tags.get("is_primary", "False")
                 )
 
+                if is_vector:
+                    dimensions = int(field.tags.get("dimensions", "0"))
+
+                    if dimensions <= 0:
```

Comment:
should we also check for the upper bound `>= 32768` ?

Reply:
yes, good idea! I'll add it